### ISSUE TITLE
Problem with Spotify linux client ?

### ISSUE THREAD

**Issue by pinkisemils** [SplittyDev/spotify-rs#1]
I'm guessing this will never work with the linux client, will it ?
As there is no option to make it open links and whatnot with the client, so it just doesn't _listen_ for things on a port - https://dacemils.plus.com/Screenshot%20from%202017-02-11%2015-05-32.png

**Comment by SplittyDev** [comment]
I'm not completely sure. I'll take a look and report back asap.

**Comment by SplittyDev** [comment]
@pinkisemils It looks like something is listening on one of the ports used by Spotify.
```sh
splitty ~ netstat -l | egrep ':437[0,1]'
tcp   0   0 localhost:4371  *:*  LISTEN
```

Could you run `netstat -l` and see if `localhost:4370` or `localhost:4371` is listed?
In the meantime I'll play around with this myself and see if I can get it to work.

**Comment by pinkisemils** [comment]
Nope, with the client app running, there's nothing listening on those ports. I'm guessing I'm running the latest version of the client I can get - 1.0.48.

**Comment by SplittyDev** [comment]
That's really strange.
I'm running `Spotify 1.0.48.103.g15edf1ec` (the official build) on ElementaryOS Loki.

When running `spoticli`, it fails to connect because it's trying to connect to port 4370.
I grabbed the latest source, changed the port to 4371 and recompiled:
```sh
splitty ~/spotify-rs ^master ~ cargo run
    Finished debug [unoptimized + debuginfo] target(s) in 0.1 secs
    Running `target/debug/spoticli`
Spotify Client (Version 1.0.48.103.g15edf1ec)
Now playing: Lemaitre - Closer
```

Could you start Spotify from the command line and post the relevant output? Maybe there is some helpful error message indicating why it doesn't listen on those ports on your machine.

**Comment by SplittyDev** [comment]
@pinkisemils I don't know if you're still interested in this, but the port issue should be fixed in 047e480.
The connector will now try to connect to both ports `4370` and `4371`.
